packages/web/src/scss/components/Tab/README.md

```diff
@@ -0,0 +1,65 @@
+# Tab
+
+The Tab component is used to organize related content. User can navigate between
+groups of information in tabbable regions.
+
+```html
+<ul class="Tab" role="tablist">
+  <li class="Tab__item">
+    <a
+      href="#pane1"
+      class="Tab__link Tab__link--selected"
+      role="tab"
+      aria-selected="true"
+      id="pane1-tab"
+      data-toggle="tab"
+      aria-controls="pane1"
+    >
+      Item Selected
+    </a>
+  </li>
+  <li class="Tab__item">
+    <a
```

Comment:
@crishpeen @adamkudrna Just for information: while handling tab switching I am not depending on any tag. So I do not care if there is `a` or `button`. Maybe because of accessibility reasons or semantics here should be used `button` instead of `a` when not using it as a link. What do you think?

Reply:
🤷 Ok, I added the button reset mixin again.